packages/toolbars-new/web/src/Toolbar/RicosToolbar.tsx

```diff
@@ -132,7 +135,7 @@ class RicosToolbar extends Component<RicosToolbarProps> {
         buttons={buttonsWithoutUnwantedSeparators}
         t={t}
         isMobile={isMobile}
-        theme={theme}
+        theme={cssOverride}
```

Comment:
Let's update prop names to our latest convention
```suggestion
        cssOverride={cssOverride}
```